main_test.go

```diff
@@ -53,11 +53,13 @@ func TestHumaRoundtrip(t *testing.T) {
 		Name:       "foo",
 		Enabled:    true,
 		Sub: &example.Sub{
-			TestEnum: example.Sub_BAR,
+			TestEnum:  example.Sub_BAR,
+			TestEnum2: 5, // Invalid value, will not serialize.
 		},
 		PrimitiveArray: []int32{1, 2, 3},
 		EnumArray: []example.Global{
 			example.Global_ONE,
+			example.Global_TWO, // Note: this is NOT public and should not be included!
```

Comment:
I'm not sure where `example.Global_TWO` is getting defined; is it supposed to be in `example.proto` above? If so, is it already defined and I'm just not seeing it?

Reply:
It's defined [in `example.proto`](https://github.com/istreamlabs/protoc-gen-huma/blob/44c3c792929c1b5c0799d10229e179b3626355bb/example.proto#L14) and excluded from the public model:

```proto
enum Global {
    NONE = 0;
    ONE = 1;
    TWO = 2 [(huma.exclude) = true];
}
```

This test case is important because we will have internal objects with invalid or non-public enum values (e.g. for internal testing or internal use-case scenarios) and we need to ensure we don't leak them.

Reply:
Ah, so we're adding it here just to prove it doesn't show up in the final rendition?